### DISCUSSION TITLE
Hiding autocomplete options

### DISCUSSION THREAD

**Discussion by Kilvoctu** [DominikDoom/a1111-sd-webui-tagcomplete#57] [Q&A] [answered]
Hi, found the extension recently and loving it. Really helpful and saves on a lot of typing, especially useful on mobile.

I'm wondering if there's any way to hide this part on the UI. I look through settings and don't see an option to show/hide this UI element.
![image](https://user-images.githubusercontent.com/2993060/199109427-239bfd47-205f-4183-85c3-377783144f93.png)
These aren't settings I'd be adjusting frequently, and it takes up a lot of UI space.

**Comment by DominikDoom** [comment, marked as answer]
I'll add a config option to hide them, didn't consider space on smaller displays.

The reason they are there in the first place is because I can't use settings in the regular webui settings tab, you *can* add new settings there but only access them on the python side as far as I'm aware, while I need them in the javascript part. Maybe I'll find a workaround in the future, but for now an option to hide them seems good.

**Reply by DominikDoom** [reply]
Added in f8a9223c29b412bf59fe1e2e620462022f5618ac, you can prevent the UI options from showing by setting `hideUIOptions` to true in the config.

**Reply by Kilvoctu** [reply]
Awesome! That works for me. Thanks so much for your work and the quick solution. 👍👍
